artist/field/actuator.py

```diff
@@ -80,7 +80,9 @@ def __init__(
         self.radius = radius
         self.phi_0 = phi_0
 
-    def motor_steps_to_angles(self, motor_steps: torch.Tensor) -> torch.Tensor:
+    def motor_steps_to_angles(
+        self, motor_steps: torch.Tensor, device: torch.device = "cpu"
```

Comment:
```suggestion
        self, motor_steps: torch.Tensor, device: Union[torch.device, str] = "gpu"
```
Technically, it is inconsistent to set a string as a default value for `device` of type `torch.device`. 
In addition, @Markus-Goetz suggested to rather set "gpu" as the default (contrary to what I told you last week).

Reply:
done